api/src/domain/tax-filings/taxFilingsInterfaceFactory.test.ts

```diff
@@ -202,6 +211,10 @@ describe("TaxFilingsInterfaceFactory", () => {
         } as TaxFilingLookupResponse);
         expect(await taxFilingInterface.onboarding({ userData, ...taxIdBusinessName })).toEqual({
           ...userData,
+          preferences: generatePreferences({
+            ...userData.preferences,
+            isCalendarFullView: true,
+          }),
```

Comment:
We've discussed tests-as-documentation before. When we add new behavior like this, there should be test(s) explicitly describing the new behavior so that someone can read the plain english and understand what the intention is. In this case, there are 3 new behaviors, so I'd expect 3 new tests that cover each case:
 a. when the lookup returns success, the isCalendarFullView gets set to true 
b. when the lookup is anything but success, and the previous value of isCalendarFullView was false, it should stay false
c. when the lookup is anything but success, and the previous value of isCalendarFullView was true, it should stay true

All these are currently tested *implicitly* in the tests you've changed, but it should **also** be described *explicitly* in its own test